js/views/accountview.js

```diff
@@ -41,9 +41,9 @@ define(function(require) {
 		},
 		ui: {
 			email: '.mail-account-email',
-			menu: 'div.app-navigation-entry-menu',
-			deleteButton: 'button[class^="icon-delete"]',
-			settingsButton: 'button[class^="icon-rename"]'
+			menu: '.app-navigation-entry-menu',
+			settingsButton: '.action-settings',
```

Comment:
Nope, because .action-settings is the whole a element which spans the full width. ;)